frontend/test/src/container/Sidebar.spec.js

```diff
@@ -1,34 +1,40 @@
-// import React from 'react'
-// import { expect } from 'chai'
-// import { mount } from 'enzyme'
-// import { Sidebar as SidebarWithoutHOC } from '../../src/container/Sidebar'
-// import sinon from 'sinon'
-// import { user } from '../hocMock/redux/user/user'
-// import { workspaceList } from '../hocMock/redux/workspaceList/workspaceList'
-// import { withRouterMockOpenContent } from '../hocMock/withRouter'
-// import { translateMock } from '../hocMock/translate'
-//
-// describe('<Sidebar />', () => {
-//   const dispatchCallBack = sinon.stub()
-//
-//   const props = {
-//     user: user,
-//     workspaceList: workspaceList.workspaceList,
-//     system: {},
-//     dispatch: dispatchCallBack,
-//     match: {
-//       params: {
-//         idws: 1,
-//         idcts: 1
-//       }
-//     }
-//   }
-//
-//   const ComponentWithHOC = withRouterMockOpenContent(translateMock()(SidebarWithoutHOC))
-//
-//   const wrapper = mount(
-//     <ComponentWithHOC {...props} />
-//   )
-// })
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import { Sidebar as SidebarWithoutHOC } from '../../../src/container/Sidebar.jsx'
+import sinon from 'sinon'
+import { user } from '../../hocMock/redux/user/user'
+import { workspaceList } from '../../hocMock/redux/workspaceList/workspaceList'
+import { withRouterMock } from '../../hocMock/withRouter'
 
-// CONTAIN DnD ITEM
+describe('<Sidebar />', () => {
+  const dispatchCallBack = sinon.stub()
+
+  const props = {
+    user: user,
+    workspaceList: workspaceList.workspaceList,
+    system: {},
+    dispatch: dispatchCallBack,
+    match: {
+      params: {
+        idws: 1,
+        idcts: 1
+      }
+    },
+    t: tradKey => tradKey
+  }
+
+  const ComponentWithHOC = withRouterMock(SidebarWithoutHOC)
+
+  const wrapper = shallow(
+    <ComponentWithHOC {...props} />
+  ).dive() // INFO - GM - 2020-03-04 - Need to dive here because we can't use mount (Dnd problems)
```

Comment:
please a bit more specific about the dnd issue.
Either some info or a link to a known issue